### PR TITLE
(#265) round default template

### PR DESCRIPTION
**Pull request by rathorevaibhav**
#265

Changes include:

1. Added migration to add two new columns in `hr_rounds` table. To store confirmed and rejected mail template.
2. View changes to save these default templates from the settings tab.
3. Showing template when the user clicks on send mail button from the application round screen.

### CODE REVIEW

File: app/Models/HR/Round.php [hunk 1]
```diff
@@ -7,10 +7,20 @@
...
+    protected $fillable = ['name', 'guidelines', 'confirmed_mail_template', 'rejected_mail_template'];
 
     protected $table = 'hr_rounds';
 
+    /**
+     * The attributes that should be cast to native types.
+     *
+     * @var array
+     */
+    protected $casts = [
```
Comment (bohraji) [inline app/Models/HR/Round.php]: 👍

File: app/Http/Controllers/SettingController.php [hunk 2]
```diff
@@ -19,7 +20,14 @@ public function index(String $module)
         $this->authorize('view', Setting::class);
 
         $settings = Setting::where('module', $module)->get()->keyBy('setting_key');
-        return view("settings.$module")->with(['settings' => $settings]);
+        return view("settings.$module")->with([
+            'settings' => $settings,
+            'rounds' => Round::all(),
+            'roundMailTypes' => [
```
Comment (bohraji) [inline app/Http/Controllers/SettingController.php]: This setting controller is very generic and deals with different modules of the whole application. Returning `rounds` info and also `round mail type` does not appear very extendible and generic.

Can we have a separate route, or route parameter or filters, etc that connects or specifies the need to return `roundMailTypes`.

**Comment by bohraji** [comment]
Checked this PR :

- [ ] For duplication.
- [ ] For performance. 
- [ ] Usage of recursive methods, loops and algorithms.
- [ ] For readability and understanding. 
- [ ] For functions with multiple responsibilities.
- [ ] For scope of refactoring. 
- [ ] For coupling
- [ ] For alternative approaches to the implementation. 
- [ ] For how the change impacts other parts of the system.
- [ ] For validations. client side and server side validations are present.

For more insights on code reviews checkout our [guidelines](https://docs.google.com/document/d/1Sa-XYJEmGwrOM5gk5zoan_Z6j90ylibmCmt331361nw/edit#).

**Comment by bohraji** [comment]
Also, @rathorevaibhav please ensure that the `travis` check have been approved before merging.

**Comment by rathorevaibhav** [comment]
@bohraji done with the changes.

**Pull request merged**